project/F84064014/README.rst

```diff
@@ -0,0 +1,101 @@
+======================================================================
+A Simple End-to-End NN Framework for Deep Learning
+======================================================================
+
+Basic Information
+=================
+
+Deep Learning is now widely used to solve a lot of problem from many different field, 
+such as computer vision, data classification, natural language processing, etc., this
+project we will implement a End-to-End Deep Learning Framework and accelerate it with C++
+
+Problem to Solve
+================
+
+In the project we try to make deep learning faster and easier
+
+Deep Learning is a powerful tool even for people who never write code before, however, 
+build a deep learning model is not a walk in the park, we hope to build a framework
+that user have to care only about which dataset they are going to use, how many layers they
```

Comment:
It is clear definition and a good starting point for designing.

As you might already anticipate, it's not easy to achieve.  You may refine the goal later.